pyintegration/deephaven2/plot/figure.py

```diff
@@ -100,15 +100,15 @@ def axes(
         name: str = None,
         axes: int = None,
         remove_series: List[str] = None,
-        plot_style: Union[str,PlotStyle] = None,
+        plot_style: Union[str, PlotStyle] = None,
     ) -> Figure:
         """TODO pydoc
```

Comment:
Looks like the pydoc comments are mostly (if not entirely) done for this.  Comment still necessary?

Reply:
yes, still missing the info on what the function does.